omaha/main.scons

```diff
@@ -70,7 +70,9 @@ _SHORT_COMPANY_NAME = 'OmahaCompanyName'
 _PRODUCT_NAME = 'Update'
 _COMPANY_DOMAIN_BASE = 'google'
 _COMPANY_DOMAIN = _COMPANY_DOMAIN_BASE + '.com'
+_SERVICE_DOMAIN = os.getenv('SERVICE_DOMAIN', "update.googleapis.com")
```

Comment:
the default value does not make much sense, since you change the path in `omaha/base/const_addresses.h` for `kUrlCrashReport`

perhaps its a better idea to have all the endpoints in `omaha/base/const_addresses.h`  as env variables (including the path segment) and fall back to the google default if they are not set